setup.cfg

```diff
@@ -12,10 +12,21 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
+    --durations 10
```

Comment:
the comment is no longer accurate, "ignore certain folders and pytest warnings". Perhaps one comment per block of things?